backend/server.js

```diff
@@ -19,7 +19,8 @@ exports.pool = mysql.createPool({
   user: process.env.RDS_USERNAME || "root",
   password: process.env.RDS_PASSWORD || "password",
   database: process.env.RDS_DB_NAME || "local_db",
-  port: process.env.RDS_PORT || "3306"
+  port: process.env.RDS_PORT || "3306",
+  multipleStatements: true
```

Comment:
This allows us to do multiple INSERT statements or SELECT calls all in one call to the database. See https://www.npmjs.com/package/mysql#multiple-statement-queries